purchase_requisition_order_remaining_qty/models/purchase_requisition_type.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class PurchaseRequisitionType(models.Model):
+    _inherit = "purchase.requisition.type"
+
+    quantity_copy = fields.Selection(
+        selection_add=(
+            [("remaining_qty", "Use remaining quantities of agreement"), ("none",)]
```

Comment:
What's that `("none", )` for?

Reply:
It's the way to insert an element before another (and the selection has that confusing value `none`):
```python
selection = [('a', 'A'), ('b', 'B')] 
selection_add = [('c', 'C'), ('b',)] 
```
> result = [('a', 'A'), ('c', 'C'), ('b', 'B')]

Reply:
Ok! It rings my bell now :)